collection/network/domain-trust.yml

```diff
@@ -0,0 +1,15 @@
+rule:
+  meta:
+    name: Detect Domain Trust Discovery 
```

Comment:
Ideally this rules completes the sentence "this function may...". How about `get domain trust relationships`?